test/monitoring_console/monitoring_console_test.go

```diff
@@ -381,7 +381,6 @@ var _ = Describe("Monitoring Console test", func() {
 		})
 	})
 
-	// TO BE READDED WHEN CSPL-1379 fixed
 	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
 		It("monitoringconsole, smoke: MC can configure SHC, indexer instances after scale up and standalone in a namespace", func() {
 			/*
```

Comment:
Nit: I don't really get what are some of these steps:
"5. Verify SH are configured in peer strings on MC Pod" What is "peer strings" here? Is it actually like "Check Search Head Pods in Monitoring Console Config Map"?
"6. Verify Monitoring Console Pod has peers in Peer string on MC Pod" Basically here it says "MC pod" twice, that's redundant.
"8. Scale Indexer Count on One Site" It's a single site cluster, "on One Site" could be removed

Reply:
These are old comments not as part of this change however good idea I can clean up.

Reply:
. Verify SH are configured in peer strings on MC Pod" What is "peer strings" here? Is it actually like "Check Search Head Pods in Monitoring Console Config Map"?
MC has `configured peer string` in one of the files thats how it configures SH's, CM, Depoyer, standalone and indexers. We check this value and compare with pods in the namespace to verify they are added to MC.

MC config map check are separate